core/src/test/resources/schema-operations-tests.adoc

```diff
@@ -281,3 +281,268 @@ directive @property(name:String) on FIELD_DEFINITION
 directive @dynamic(prefix:String = "properties.") on FIELD_DEFINITION
 
 ----
+
+=== Schema with capitalized queries
+
+.Configuration
+[source,json,schema-config=true]
+----
+{
+  "capitalizeQueryFields": true
+}
+----
+
+.Augmented Schema
+[source,graphql]
+----
+schema {
+  query: CustomQueryType
+  mutation: Mutation
+}
+
+type CustomQueryType {
+  Location(filter: _LocationFilter, first: Int, name: String, offset: Int, orderBy: [_LocationOrdering!]): [Location!]!
+  Person(filter: _PersonFilter, first: Int, name: String, offset: Int, orderBy: [_PersonOrdering!]): [Person!]!
+  person(name: String): Person @cypher(statement : "MATCH (p:Person { name:{name} }) RETURN p")
+}
+
+type Location {
+  name: String
+}
+
+type Mutation {
+  createLocation(name: String): Location!
+  createPerson(name: String): Person!
+}
+
+type Person {
+  name: String
+}
+
+type _Neo4jDate {
```

Comment:
could we just out-source that and use an `include::neo4j-types.adoc[]` here (and elsewhere where we check the augmented schema)?